edflow/explore.py

```diff
@@ -1,33 +1,38 @@
 import os
 import sys
+from typing import Any
```

Comment:
so far we haven't used much type hints. I'm not strongly opposed but maybe you can weigh up if we want to introduce type hints here at the cost of introducing inconsistent coding style.

Reply:
removed the type hints. They are still there in the docstrings.